server/runner/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -2,7 +2,7 @@ smallrye.graphql.printDataFetcherException=true
 smallrye.graphql.tracing.enabled=true
 smallrye.graphql.allowGet=true
 smallrye.graphql.logPayload=true
-
 mp.graphql.showErrorMessage=java.security.AccessControlException,io.smallrye.graphql.test.apps.exceptionlist.*
 mp.graphql.hideErrorMessage=java.io.IOException,io.smallrye.graphql.test.apps.exceptionlist.*
-smallrye.graphql.errorExtensionFields=exception,classification,code,description,validationErrorType,queryPath
\ No newline at end of file
+smallrye.graphql.errorExtensionFields=exception,classification,code,description,validationErrorType,queryPath
+smallrye.graphql.federation.enabled=true
```

Comment:
Maybe we should make the default `true` ? I might have said the opposite in the past, but looking at this now, I think this can be enabled by default ? We not breaking anything with the current usage, right ?

Reply:
not that I'm aware of. will do